src/plone/restapi/serializer/dxcontent.py

```diff
@@ -62,11 +63,14 @@ def __call__(self, version=None, include_items=True):
         parent_summary = getMultiAdapter(
             (parent, self.request), ISerializeToJsonSummary
         )()
+        parent_summary["type_name"] = get_portal_type(parent.portal_type, self.request)
+
```

Comment:
`type_name` is included in the summary serializer, so I don't think it needs to be added here.